send_to_s3.py

```diff
@@ -27,15 +29,17 @@ def eprint(*args, **kwargs):
 
     for root, subFolders, files in os.walk(sourcedir):
         for file in files:
-            filename = os.path.join(root,file)
+            filename = os.path.join(root, file)
             filepath = "{0}/{1}".format(filedir, os.path.relpath(filename, sourcedir))
             print("{0} => {1}".format(filename, filepath), file=sys.stderr)
             key = boto.s3.key.Key(bucket, filepath)
-            file_to_send = open(filename, 'r')
+            file_to_send = open(filename, 'rb')
```

Comment:
Open file in binary mode.  It appears this works for all file types encountered, but if something fails, I'll do this more methodically, and only read in binary mode for certain file types by extension.